web-ui/src/main/resources/catalog/components/common/needhelp/NeedHelpDirective.js

```diff
@@ -69,7 +69,7 @@
 
           var helpBaseUrl =
             gnGlobalSettings.docUrl ||
-            "https://geonetwork-opensource.org/manuals/trunk/{lang}";
+            "https://docs.geonetwork-opensource.org/latest/{lang}";
```

Comment:
Is there any interaction between this docUrl setting and the inclusion of online help in the war? 

Previously the build system had logic to include the entire geonetwork user manual, now it only includes the online help section for visitors.  Perhaps an additional setting would be useful.

Reply:
Afaik, the online docs included in the war are not used anymore.